### PR TITLE
Update quick start guide based on feedback

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Implement feedback from https://github.com/mattermost/mattermost-developer-documentation/pull/806

#### Ticket Link
None

### CODE REVIEW

File: site/content/integrate/apps/quick-start-go/_index.md [hunk 1]
```diff
@@ -49,7 +50,10 @@ go get github.com/mattermost/mattermost-plugin-apps/apps@master
 
 ### Manifest
 
-Your app has to provide a so-called manifest. The manifest declares app metadata. In this example the *permission* to act as a bot, and to *bind* itself to the channel header, and to `/` commands is requested.
+Your app has to provide a manifest, which declares app metadata. In this example, the following permissions are requested:
+- Create posts as a bot.
+- Render icons in the channel header.
```
Comment (mickmister) [inline site/content/integrate/apps/quick-start-go/_index.md:55]:
```suggestion
- Render icons in the channel header that will communicate with your app when clicked.
```

File: site/content/integrate/apps/quick-start-js/_index.md [hunk 2]
```diff
@@ -1,28 +1,26 @@
...
 
 ## Prerequisites
 
-Before you can start with your app, you first need to set up a local developer environment following the [server](/contribute/server/developer-setup/) and [webapp](/contribute/webapp/developer-setup/) setup guides. You must enable the apps feature flag before starting the Mattermost server by setting the enjoinment variable `MM_FEATUREFLAGS_AppsEnabled` to `true` by e.g. adding `export MM_FEATUREFLAGS_AppsEnabled=true` to your `.bashrc`. Please also ensure that `Bot Account Creation` and `OAuth 2.0 Service Provider` are enabled in the System Console.
+Before you can start with your app, you first need to set up a local developer environment following the [server](/contribute/server/developer-setup/) and [webapp](/contribute/webapp/developer-setup/) setup guides. You must enable the apps feature flag before starting the Mattermost server by setting the environment variable `MM_FEATUREFLAGS_AppsEnabled` to `true` by e.g. adding `export MM_FEATUREFLAGS_AppsEnabled=true` to your `.bashrc`. Please also ensure that `Bot Account Creation` and `OAuth 2.0 Service Provider` are enabled in the System Console.
```
Comment (mickmister) [inline site/content/integrate/apps/quick-start-js/_index.md:17]: I think someone that doesn't want to pollute their `.bashrc` file may want to know the alternative of

`make run-server MM_FEATUREFLAGS_AppsEnabled=true`

**Review by mickmister — COMMENTED** [review]
Thanks @hanzei, great work :+1: 

I found one small improvement. Will re-review the js portion once https://github.com/mattermost/mattermost-plugin-apps/pull/174 is merged

File: site/content/integrate/apps/quick-start-js/_index.md [hunk 3]
```diff
@@ -31,115 +29,114 @@ git clone https://github.com/mattermost/mattermost-plugin-apps.git
...
 
 ## Building the app
 
-Start building your app by creating a directory for the code:
+Start building your app by creating a directory for the code, setup a new package and install  [`node-fetch`](https://www.npmjs.com/package/node-fetch) and [`express`](https://expressjs.com/), which will be used later:
 
 ```bash
 mkdir my-app
 cd my-app
+npm init
```
Comment (mickmister) [inline site/content/integrate/apps/quick-start-js/_index.md:45]: Maybe suggest `npm init -y` to avoid the prompts with running this command

Comment (hanzei) [reply site/content/integrate/apps/quick-start-js/_index.md:45]: I'm hesitant to use `-y` as the default suggested way. A dev can just spam enter anyway.

**Review by justinegeffen — APPROVED** [review]

File: site/content/integrate/apps/quick-start-js/_index.md (hunk 2 above)
Comment (mickmister) [reply site/content/integrate/apps/quick-start-js/_index.md:17]: Is this something you meant to add? This thread is marked as resolved but I don't see the `make run-server` example in the PR

**Comment by mickmister** [comment]
Do we want to merge this now and update the js example again after https://github.com/mattermost/mattermost-plugin-apps/pull/174 is merged?

File: site/content/integrate/apps/quick-start-js/_index.md (hunk 2 above)
Comment (hanzei) [reply site/content/integrate/apps/quick-start-js/_index.md]: Forgot to push. It's now in: f4c28c0b54a122d93c7169991c6de39137539692

**Comment by hanzei** [comment]
@mickmister Nah, let's wait https://github.com/mattermost/mattermost-plugin-apps/pull/174 to be merged.

**Review by mickmister — APPROVED** [review]

**Comment by mickmister** [comment]
One request before merging. The js example file in the apps repo should be linked toward the top of the documentation page

**Comment by justinegeffen** [comment]
> One request before merging. The js example file in the apps repo should be linked toward the top of the documentation page

I've added that link - LMK if that's the right place. 

Are we still waiting on https://github.com/mattermost/mattermost-plugin-apps/pull/174 to be merged before we merge this?

**Pull request merged**